tests/rkt_version_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+// Check the version of the Go compiler used to compile rkt
+func TestVersion(t *testing.T) {
```

Comment:
Good idea.

I would still like to have a functional test for it in the context of installed tests: https://github.com/coreos/rkt/issues/1720